src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplDownloadSteps.java

```diff
@@ -34,13 +62,32 @@ public ChplDownloadSteps() {
     }
 
     /**
-     * Loads the CHPL Download page.
+     * Get user to the Download CHPL page. Chrome options are necessary to get past
+     * keep/discard pop ups for successful download of a file to directory.
+     * @throws Throwable throws exception if there is an issue with Chrome options.
      */
-    @Given("^user is on CHPL download page$")
-    public void userIsOnCHPLdownloadPage() {
+    @Given("^I am on download the CHPL resources page$")
+    public void iAmOnDownloadTheCHPLResourcesPage() throws Throwable {
+        HashMap<String, Object> chromePrefs = new HashMap<String, Object>();
+        chromePrefs.put("profile.default_content_settings.popups", 0);
+        chromePrefs.put("download.default_directory", downloadPath);
+        /**
+         * Save Chrome Options
+         */
+        ChromeOptions options = new ChromeOptions();
+        HashMap<String, Object> chromeOptionsMap = new HashMap<String, Object>();
+        options.setExperimentalOption("prefs", chromePrefs);
+        chromePrefs.put("safebrowsing.enabled", "true");
+        options.addArguments("--safebrowsing-disable-download-protection");
+        options.addArguments("disable-popup-blocking");
+
+        DesiredCapabilities cap = DesiredCapabilities.chrome();
+        cap.setCapability(ChromeOptions.CAPABILITY, chromeOptionsMap);
+        cap.setCapability(CapabilityType.ACCEPT_SSL_CERTS, true);
+        cap.setCapability(ChromeOptions.CAPABILITY, options);
+
+        driver = new ChromeDriver(cap);
         driver.get(url + "#/resources/download");
-        WebDriverWait wait = new WebDriverWait(driver, TIMEOUT);
```

Comment:
Should bring this back. With the long load time of our pages I think it's a good idea to have the "load the page" step not return until the page is actually fully loaded